llvm/test/CodeGen/AArch64/naked-fn-with-frame-pointer.ll

```diff
@@ -0,0 +1,39 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple aarch64 | FileCheck %s -check-prefixes=NAKED-LE
+; RUN: llc < %s -mtriple aarch64_be | FileCheck %s -check-prefixes=NAKED-BE
+
+declare dso_local void @main()
+
+define dso_local void @naked() naked "frame-pointer"="all" {
+; NAKED-LE-LABEL: naked:
```

Comment:
Nit: since we are now also checking non-naked functions you could to a search-replace of `NAKED-` -> `CHECK-` in this diff. But not particularly important.

Reply:
Fair point; done.